src/compas/scene/meshobject.py

```diff
@@ -106,16 +106,6 @@ def mesh(self, mesh):
         self._item = mesh
         self._transformation = None
 
-    @property
-    def transformation(self):
-        # type: () -> compas.geometry.Transformation | None
-        return self._transformation
-
-    @transformation.setter
-    def transformation(self, transformation):
-        # type: (compas.geometry.Transformation) -> None
-        self._transformation = transformation
-
```

Comment:
Same here, this is a breaking change, and maybe even not that small. It's not clear why this is needed, and what's the new way of doing the same thing if this property goes away

Reply:
this is just duplication form functionality of the base class

Reply:
the parent already defines it so it should not have been here to begin with. functionality will not change...

Reply:
Ah! Got it!